onboarding/scaffolding.py

```diff
@@ -229,7 +242,7 @@ def perform_mutate_for_authenticated_user(cls, form, info: ResolveInfo):
 
     @classmethod
     def perform_mutate_for_anonymous_user(cls, form, info: ResolveInfo):
-        update_scaffolding(info.context, form.cleaned_data)
+        update_scaffolding(info.context, get_scaffolding_fields_from_form(form))
         return cls.mutation_success()
```

Comment:
Note that we don't actually _need_ to make this change--none of the mutations that subclass `OnboardingScaffoldingOrUserDataMutation` actually have scaffolding fields that need renaming--but I thought it appropriate to have the same kind of behavior in all our onboarding scaffolding mutation classes for consistency.

Besides, I suspect we will move more of our `OnboardingScaffoldingMutation`s to `OnboardingScaffoldingOrUserDataMutation`s over time, and making their behavior consistent now will reduce surprises in the future.